apps/editors/helpers.py

```diff
@@ -476,8 +478,13 @@ def get_actions(self, request, addon):
         labels, details = self._review_actions()
 
         actions = SortedDict()
-        if not addon.admin_review or acl.action_allowed(
-                request, 'ReviewerAdminTools', 'View'):
+        if ((not addon.admin_review or acl.action_allowed(request,
+             'ReviewerAdminTools', 'View'))
+            and (not GroupUser.objects.filter(group=Group.objects.get(
+                 name='Limited Reviewers'), user=request.user)
```

Comment:
There is a Django shortcut for this (which also optimizes the join): `GroupUser.objects.filter(group__name='Limited Reviewers', user=...)`. The double underscore does the join for you.